### PR TITLE
left old rides API as accept and clone new one as scheduled

### PR DESCRIPTION
**Pull request by Adlife84**
## Description
CR-112 (Update)
Clone rides API endpoint to say scheduled not accepted

What does the PR do?
clone rides API from accept to scheduled and left both API because somebody still use old request 'accept' instead 'scheduled'

What does the PR do?

## Screenshots
Screen Shot 2021-03-10 at 2 20 21 PM

## Checklist:
- [ ] Screenshots attached (if dealing with UI)
- [ ] Unit Tests (if applicable)

## Testing Steps for reviewer:
- Step 1
- Step 2

@micronix @kidatsy @jrmcgarvey

### CODE REVIEW

**Review by jrmcgarvey — COMMENTED** [review]
Hm -- I could approve this, but it bothers me that the post for accept and the post for scheduled duplicate about 20 lines of code.  I think it would be better if to DRY the code so that it is not repeated.

File: app/controllers/api/v1/rides.rb [hunk 1]
```diff
@@ -124,12 +124,12 @@ class Rides < Grape::API
         end
       end
 
-      # Driver Acceting only an approved ride
+      # Driver Acceting only an approved ride with old API "accept"
```
Comment (kidatsy) [inline app/controllers/api/v1/rides.rb:127]: Can you elaborate on this comment, to note that we will be removing this endpoint once we have https://github.com/CodeTheDream/ride-share-app/pull/115 pushed to production and distributed?

**Review by kidatsy — COMMENTED** [review]
LGTM, could just use some more detail in the comment on the old `/accept` endpoint

**Review by jrmcgarvey — APPROVED** [review]
ok

**Pull request merged**